bindings/python/setup.py

```diff
@@ -2,15 +2,69 @@
 # encoding: utf-8
 
 from setuptools import setup, Extension
+from pathlib import Path
+import os
+import shutil
+import sys
+import subprocess
+from distutils.command.build import build
+from distutils.command.sdist import sdist
```

Comment:
```suggestion
from setuptools.command.sdist import sdist
```
The distutils package is deprecated